modules/update-restricted-network-cli.adoc

```diff
@@ -0,0 +1,186 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-restricted-network.adoc
+
+[id="update-restricted-network-cli_{context}"]
+= Update the restricted network cluster
+
+After you update the contents of the mirror registry, you can update your
+restricted network cluster.
+
+.Prerequisites
+
+* You have access to the mirror registry that you used to store the images that
+you used to install {product-title}.
+* You have the `ImageContentSourcePolicy` information for the content that you
+mirrored.
+* The the OpenShift Command-line Interface (CLI), commonly known as `oc`, is
+installed.
+* The `jq` package is installed.
+
+.Procedure
+
+Complete the following steps on the bastion host:
+
+. Review the `ImageContentSourcePolicy` resource for your cluster:
++
+----
+$ oc get ImageContentSourcePolicy -o yaml
+
+apiVersion: operator.openshift.io/v1alpha1
+kind: ImageContentSourcePolicy
+metadata:
+  name: example
+spec:
+  repositoryDigestMirrors: <1>
+  - mirrors:
+    - <local_registry_host_name>:<local_registry_host_port>/<repository_name>/release
+    source: quay.io/openshift-release-dev/ocp-release
+  - mirrors:
+    - <local_registry_host_name>:<local_registry_host_port>/<repository_name>/release
+    source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
+----
+<1> The `repositoryDigestMirrors` section must contain the `mirrors` data that
+is in the output of the `oc adm mirror release` command from when you mirrored
+the images for this release.
+
+. If the `repositoryDigestMirrors` section of the `ImageContentSourcePolicy` does not contain the `mirrors` for this release, create another `ImageContentSourcePolicy` resource:
+.. Record the name of the current `ImageContentSourcePolicy`. You will delete this resource after you complete the update.
+.. Define the `ImageContentSourcePolicy` resource for your cluster:
++
+----
+$ cat test.yaml
+apiVersion: operator.openshift.io/v1alpha1 <1>
+kind: ImageContentSourcePolicy
+metadata:
+  name: example
+spec:
+  repositoryDigestMirrors:
+  - mirrors:
+    - <local_registry_host_name>:<local_registry_host_port>/<repository_name>/release
+    source: quay.io/openshift-release-dev/ocp-release
+  - mirrors:
+    - <local_registry_host_name>:<local_registry_host_port>/<repository_name>/release
+    source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
+----
+<1> Specify the entire specification that was showin in the output of the
+`oc adm mirror release` command that mirrored the images for this release.
+.. Create the `ImageContentSourcePolicy` resource that you defined:
++
+----
+$ oc create -f test.yaml
+----
+.. Confirm that the new `ImageContentSoucePolicy` is in effect by reviewing the node status:
```

Comment:
How about adding some words like - "Wait for some mins, then confirm that the new `ImageContentSoucePolicy` is in effect"